spec/s3_text_file_spec.rb

```diff
@@ -60,11 +60,19 @@
   it 'overwrites a file' do
     with_temp_bucket(files: [{key: key, path: test_file_path}]) do
       instance = described_class.new(bucket_name: bucket_name, bucket_region: region, key: key)
-      instance.write("something new")
+      instance.write(string_contents: "something new")
       expect(instance.read).to eq "something new"
     end
   end
 
+  it 'sets the cache control header' do
+    with_temp_bucket(files: [{key: key, path: test_file_path}]) do
+      instance = described_class.new(bucket_name: bucket_name, bucket_region: region, key: key)
```

Comment:
optional of course.  this could be cleaned up a bit via a subject http://www.betterspecs.org/#subject

Reply:
Added a `let`